tasks/build-jsr.ts

```diff
@@ -0,0 +1,15 @@
+import jsonDeno from "../deno.json" with { type: "json" };
+
+await Deno.writeTextFile(
+  new URL("../deno.json", import.meta.url),
+  JSON.stringify({
+    ...jsonDeno,
+    name: "@effection/effection",
+    version: Deno.env.get("VERSION"),
```

Comment:
We can include everything in the source `deno.json` except for the version since it will never change, right?

Reply:
Done.